Sources/StreamChatUI/Components.swift

```diff
@@ -70,12 +70,15 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     /// The view used to display content of the message, i.e. in the channel detail message list.
     public var messageContentView: _ChatMessageContentView<ExtraData>.Type = _ChatMessageContentView<ExtraData>.self
 
-    /// The injector used to inject gallery attachment views
+    /// The injector used to inject gallery attachment views.
     public var galleryAttachmentInjector: _AttachmentViewInjector<ExtraData>.Type = _GalleryAttachmentViewInjector<ExtraData>.self
```

Comment:
👏 😅 Happy to see you became friends with punctuation.

Reply:
indeed 😄